src/orch/orch.c

```diff
@@ -1,7 +1,47 @@
+#include "../common/dbus.h"
+#include "controller.h"
+
 #include <stdio.h>
 #include <stdlib.h>
 
-int main() {
+int getArgs(int argc, char *argv[], int *ctrlPort) {
```

Comment:
Use snake_case here for consistency with systemd: get_args().  Also, consider using getopt().

Reply:
Changed the naming and tried to implement it via `getopt()`. Moved it to a dedicated file. If this is ok, I'd try to implement basic tests for it in a follow-up PR.